src/workspaces/features/WorkspaceMembers/WorkspaceMembers.tsx

```diff
@@ -0,0 +1,78 @@
+import Button from "core/components/Button";
+import DataGrid, { BaseColumn } from "core/components/DataGrid";
+import DateColumn from "core/components/DataGrid/DateColumn";
+import { TextColumn } from "core/components/DataGrid/TextColumn";
+
+import { DateTime } from "luxon";
+import { useTranslation } from "react-i18next";
+import { useWorkspacePageQuery } from "workspaces/graphql/queries.generated";
```

Comment:
You need to define your query for this component and not use the one from the workspace page.